backend_kvm.go

```diff
@@ -74,14 +75,58 @@ func (b kvmBackend) hostKernelRelease() (string, error) {
 	return "", fmt.Errorf("No kernel found")
 }
 
+func (b kvmBackend) hostKernelPath(kernelRelease string) (string, error) {
+	kernelDir := "/boot"
```

Comment:
Kernel not always reside in `/boot`. For exmple, on systemd-boot using hosts, kernels are likely to end up in `/boot/efi/$machine_id/$uname_r/linux` (or `/efi`). On Fedora, `kernel` packages install the kernel at `/usr/lib/modules/$uname_r/vmlinuz`.
I guess supporting all the cases might be a lot of work, so maybe it's possible make the kernel path selectable manually via a CLI arg or an environment variable?

Reply:
Since the existing code already assumes `/boot`, patch as-is is a good start. IIRC most distros install the kernel image in `/usr/lib/modules/$uname_r/vmlinuz` while the `/boot` or `/efi` one being a copy, during the package post-install stage.

Reply:
> install the kernel image in `/usr/lib/modules/$uname_r/vmlinuz` while the `/boot` or `/efi` one being a copy

That's accurate, at least for Fedora.